lib/global.dart

```diff
@@ -0,0 +1,7 @@
+library concordi_around.globals;
```

Comment:
Will most likely be replacing globals with a DisabilityNotifier

Reply:
I agree with your comment